NAMESPACE

```diff
@@ -1,16 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
 export(agg)
-export(assert_no_missing_intervals)
-export(assert_no_overlapping_intervals)
+export(assert_no_missing_intervals_dt)
+export(assert_no_overlapping_intervals_dt)
 export(collapse_common_intervals)
 export(create_agg_tree)
 export(create_scale_tree)
 export(gen_end)
 export(gen_length)
 export(gen_name)
-export(identify_missing_intervals)
-export(identify_overlapping_intervals)
+export(identify_missing_intervals_dt)
```

Comment:
Does the function need to change names?

Reply:
Its not changing names. `identify_missing_intervals` is the same, just no longer exported from the package. `identify_missing_intervals_dt` is a new function that will be exported